README.md

```diff
@@ -2,6 +2,12 @@
 
 [![CircleCI](https://circleci.com/gh/remove-bg/go.svg?style=shield)](https://circleci.com/gh/remove-bg/go)
 
+Run the test locally. Workflows cannot be run locally!
+
+```
+circleci local execute --job test
+```
+
```

Comment:
Since it's a public repository and contributors have no access to our CI environment, I would rather provide instructions for running the tests truly locally.